src/components/Time/DeltaTime.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+    <div class="DeltaTime">
+        +{{ prettyTime }}s
+    </div>
+</template>
+
+<script>
+import moment from 'moment';
+
+export default {
+    name: 'DeltaTime',
+    props: {
+        time: Number,
+        firstTime: Number,
+        bonusSeconds: String
+    },
+    computed: {
+        prettyTime() {
+            const firstplace = Math.floor(this.firstTime * 10);
+            const yourtime =  Math.floor(this.time * 10);
+            const delta = Math.round((yourtime - firstplace)) / 10;
+
+            const momentTime = moment
+                .unix(delta + +this.bonusSeconds)
+                .utc();
+
+            if (momentTime.utc().minutes()) {
+                if (momentTime.utc().minutes() < 10) return momentTime.format('M.ss.S');
+                else return momentTime.format('MM.ss.S');
+            } else {
+                if (momentTime.seconds() < 10) return momentTime.format('s.S');
+                return momentTime.format('ss.S');
+            }
```

Comment:
Use brackets and the else is not needed.
```suggestion
            if (momentTime.utc().minutes()) {
                if (momentTime.utc().minutes() < 10) {
                    return momentTime.format('M.ss.S');
                }
                return momentTime.format('MM.ss.S');
            }
            if (momentTime.seconds() < 10) {
                return momentTime.format('s.S');
            }
           momentTime.format('ss.S');
```

Reply:
Not sure if the syntax is completly correcy

Reply:
I do see monentjs has an trim option

Reply:
Waarom doen jullie eigenlijk reviews in t Engels als iedereen Nederlands is?

Reply:
Ik denk omdat de code in het engels is.